charts/seed-monitoring/charts/prometheus/rules/apiserver-connectivity-check.rules.yaml

```diff
@@ -0,0 +1,24 @@
+groups:
+- name: apiserver-connectivity-check.rules
+  rules:
+  - alert: APIServerUnreachableFromShoot
+    expr: probe_success{job=~"blackbox-exporter-k8s-service-check-[0-9]"} != 1
+    for: 5m
+    labels:
+      service: apiserver-connectivity-check
+      severity: critical
+      type: shoot
+    annotations:
+      summary: API Server is not reachable from shoot.
+      description: The API Server is unreachable from the shoot.
+
+  - alert: ConnectionToAPIServerSlow
+    expr: avg(probe_duration_seconds{job=~"blackbox-exporter-k8s-service-check-[0-9]"}) by (instance) > 0.03
+    for: 5m
+    labels:
+      service: apiserver-connectivity-check
+      severity: warning
+      type: shoot
+    annotations:
+      summary: Connection to API Server from shoot is slow.
+      description: The average connection to the API Server from the shoot is over 0.03s.
```

Comment:
We need to adapt the summaries, descriptions and names of the alerts to make clear what action could not be completed and what's the impact. 

For example:
```
 - alert: ApiServerTcpConnectionTimeout
  ....
  annotations:
      summary: ApiServer tcp connection timeout
      description: At least one of XXX tries to establish a tcp connection to the Api server via the infrastructure load balancer failed after 128 seconds timeout.
```

Reply:
done